R/FilterState.R

```diff
@@ -116,8 +120,9 @@ FilterState <- R6::R6Class( # nolint
       private$selected <- reactiveVal()
       private$keep_na <- if (is.null(keep_na) && anyNA(x)) reactiveVal(TRUE) else reactiveVal(keep_na)
       private$keep_inf <- reactiveVal(keep_inf)
+      private$disabled <- if (isTRUE(locked)) reactiveVal(FALSE) else reactiveVal(disabled)
       private$fixed <- fixed
-      private$disabled <- reactiveVal(disabled)
+      private$locked <- locked
```

Comment:
I hope you meant `Since locked can override disabled`? I would not put much energy in raising any error as the whole disabled functionality is about to be removed with #300